auto_round/mllm/README.md

```diff
@@ -25,13 +25,13 @@ AutoRound uses the text module of MLLM (LLM component) as the main quantization
     autoround.save_quantized(output_dir, format='auto_round', inplace=True)
 ```
 
-- `dataset`: the dataset for quantization training. current support NeelNanda/pile-10k,llava_conv_58k,llava_instruct_80k. It can be a custom one.
+- `dataset`: the dataset for quantization training. default to NeelNanda/pile-10k. For llava dataset, currently limited support llava_conv_58k, llava_instruct_80k and llava_instruct_150k. It can be a custom one.
```

Comment:
the limitation is not for dataset, but for models, we only support llava dataset for 5 models, right? Since we haven't verify the default template is ok or not?